apps/user-office-frontend/src/components/proposal/ProposalTableInstrumentScientist.tsx

```diff
@@ -188,18 +243,26 @@ const ProposalTableInstrumentScientist = ({
     questionFilter: questionaryFilterFromUrlQuery(urlQueryParams),
     reviewer: getFilterReviewer(urlQueryParams.reviewer),
   });
+  const [queryParameters, setQueryParameters] = useState<QueryParameters>({
+    first: 2000,
+  });
+  const [maximumToload, setMaximumToload] = useState(2000);
```

Comment:
I am not quite sure why we need two states that keep the same value. As I see the `queryParameters` has property `first` which keeps the same value as `maximumToLoad`. I was thinking if we can reuse the `queryParameters.first` everywhere instead. Maybe I am wrong here and this is not possible but you can at least give it a try.